docs/tutorials/image_signing.md

```diff
@@ -0,0 +1,189 @@
+# How to sign and distribute container images using Podman
+
+Signing container images originates from the motivation of trusting only
+dedicated image providers to mitigate man-in-the-middle (MITM) attacks. One way
+to sign images is to utilize a GNU Privacy Guard ([GPG][0]) key. This technique
+is generally compatible with any OCI compliant container registry like
+[Quay.io][1]. It is worth mentioning that the OpenShift integrated container
+registry supports this signing mechanism out of the box.
+
+[0]: https://gnupg.org
+[1]: https://quay.io
+
+From a technical perspective, we can utilize Podman to sign the image before
+pushing it into a remote registry. After that, all systems running Podman have
+to be configured to retrieve the signature key from a remote server, which can
+be any simple web server. This means that every unsigned image will be rejected
+during an image pull operation. But how does this work?
+
+First of all, we have to create a GPG key pair or select an already locally
+available one:
+
+```bash
+> gpg --list-keys sgrunert@suse.com
+pub   rsa2048 2018-11-26 [SC] [expires: 2020-11-25]
+      92836C5387398A449AF794CF8CE029DD1A866E52
+uid           [ultimate] Sascha Grunert <sgrunert@suse.com>
+sub   rsa2048 2018-11-26 [E] [expires: 2020-11-25]
+```
```

Comment:
If you have not, I'd change the above values in your key just to be ultra safe.

Reply:
Fixed :+1:

Reply:
(Aren’t they the public part of the key, which every consumer must have to be able to verify the signatures? No reason to undo the edit now, of course.)